packages/documentation/src/content/docs/integration/prod/docker-compose.mdx

```diff
@@ -123,27 +348,29 @@ server {
 }
 
 server {
-    server_name auth.myrafiki.com;
+    server_name ilp.myrafiki.com;
 
     listen 80;
 
-    if ($host = auth.myrafiki.com) {
-	    return 301 https://$host$request_uri;
+    if ($host = ilp.myrafiki.com) {
+            return 301 https://$host$request_uri;
```

Comment:
We should standardize the tabs/spaces of the nginx files, I recommend one indentation = 4 spaces